OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -18,9 +29,9 @@ inline void clock_gettime(uint32_t, struct timespec* spec) {
 }
 #endif
 
-inline TickType OrbitTicks(uint32_t a_Clock = 1 /*CLOCK_MONOTONIC*/) {
+inline TickType MonotonicTimestampNs() {
   timespec ts;
-  clock_gettime(a_Clock, &ts);
+  clock_gettime(CLOCK_MONOTONIC, &ts);
```

Comment:
Why do you want to use monotonic here? If this is because of performance maybe consider using CLOCK_BOOTTIME on linux?

Reply:
It's the clock used by the perfeventopen events.  I'm not changing the behavior, just shuffling code around.